source/_patterns/03-templates/55-monthly-good/01-landing.mustache

```diff
@@ -2,7 +2,11 @@
 	<div class="columns monthly-good monthly-good-landing">
 		<p>Monthly Good <strong>landing</strong> page placeholder</p>
 
-		<p><a class="button" href="{{confirm_href}}">Join Monthly Good</a></p>
+		<p>
+			<div class="mg-deposit-amount-input-wrapper">
+				<input class="mg-deposit-amount-input" type="text" maxlength="9" placeholder="$" value="$25" data-abide-validator="currency" data-min-amount="0.00" data-max-amount="10000" required="">
+			</div>
+		<a class="button join-monthly-good" href="{{confirm_href}}">Join Monthly Good</a></p>
```

Comment:
eventually, we'll want to manage this link via javascript, but this is fine for now.